cypress/integration/patch-route.js

```diff
@@ -161,10 +161,78 @@ describe("Patch route", function() {
           clickSorterAndAssertTasksAreFetched(sortBy)
         );
       });
+
+      it("Fetches additional tasks as the user scrolls", () => {
+        cy.get(".ant-table-row")
+          .invoke("toArray")
+          .then($initialTasks => {
+            cy.get(".ant-table-body").scrollTo("bottom", { duration: 300 });
+            cy.wait(200);
+
+            waitForGQL("@gqlQuery", "PatchTasks");
+            cy.get("@gqlQuery")
+              .its("requestBody.variables.page")
+              .should("equal", 1);
+
+            // confirm additional tasks were appended to table items
+            cy.get(".ant-table-row").should(
+              "have.length.greaterThan",
+              $initialTasks.length
+            );
+          });
+      });
+
+      it("Task count increments by the number of additional tasks fetched", () => {
+        waitForGQL("@gqlQuery", "PatchTasks");
+        cy.get("[data-cy=current-task-count]")
+          .invoke("text")
+          .then($initialTaskCount => {
+            cy.get(".ant-table-body").scrollTo("bottom", { duration: 300 });
+            cy.wait(200);
+            waitForGQL("@gqlQuery", "PatchTasks");
+
+            cy.get("[data-cy=current-task-count]")
+              .invoke("text")
+              .then($newTaskCount => {
+                expect(parseInt($newTaskCount)).is.gt(
+                  parseInt($initialTaskCount)
+                );
+              });
+          });
+      });
+
+      it("Stops fetching tasks when all tasks have been fetched", () => {
+        scrollTasksTableUntilAllTasksFetched({ hasMore: true });
+        cy.get(".ant-table-body").scrollTo("bottom", { duration: 300 });
```

Comment:
there is a chance that the table has fully scrolled to the bottom by line 206. It may be a better to scroll all the way up and all the way down again to confirm no additional data will be fetched from scrolling.